packages/react-graphql/src/hooks/utilities.ts

```diff
@@ -0,0 +1,31 @@
+import isPlainObject from 'lodash/isPlainObject';
```

Comment:
_carry over things that's not resolved yet but lost due to moving files_

https://github.com/Shopify/quilt/pull/642#discussion_r273723029
> No Lodash dependencies 🙈